src/routes/index.ts

```diff
@@ -13,11 +14,16 @@ export function registerRoutes(router: Router): void {
   const cachedR2Middleware = cached(r2Middleware);
   const originMiddleware = new OriginMiddleware();
 
+  const corepackRedirectMiddleware = new RedirectionMiddleware('https://github.com/nodejs/corepack');
```

Comment:
```suggestion
  const corepackRedirectMiddleware = new RedirectionMiddleware('https://github.com/nodejs/corepack#-corepack');
```

So it highlights the README?

Reply:
Would this not be better so its not reliant on the first heading not changing?
```suggestion
  const corepackRedirectMiddleware = new RedirectionMiddleware('https://github.com/nodejs/corepack#readme');
```

Reply:
Oh cool, nice that thbis works!